demo-pipeline/terraform/codebuild.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_codebuild_project" "steps" {
+  for_each = toset(local.codebuild_projects)
+
+  name           = each.value
+  service_role   = aws_iam_role.codebuild_service[each.value].arn
+  description    = local.project_properties[each.value]["description"]
+  encryption_key = "arn:aws:kms:us-east-2:489999593185:alias/aws/s3"
```

Comment:
I assume this hard-coded key will only work in your account right?

Reply:
It's the default Amazon KMS S3 key, not customer managed.  It will work everywhere.